src/ontology/mondo-ingest.Makefile

```diff
@@ -13,6 +13,28 @@
 MAPPINGSDIR=../mappings
 SKIP_HUGE=false
 
+####################################
+### Package management #############
+####################################
+.PHONY: pip
+pip:
+	python3 -m pip install --upgrade pip setuptools
+
+.PHONY: oaklib
+oaklib: pip
```

Comment:
Added this, as I have need for latest `oaklib` this right now in some of my work.

Reply:
ok